spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingFilterRowsWithErrors.scala

```diff
@@ -0,0 +1,44 @@
+package za.co.absa.spark.commons.errorhandling.implementations
+
+import org.apache.spark.sql.{Column, DataFrame}
+import org.apache.spark.sql.functions.{coalesce, lit}
+import za.co.absa.spark.commons.errorhandling.ErrorMessageSubmit
+import za.co.absa.spark.commons.errorhandling.partials.ErrorHandlingCommon
+import za.co.absa.spark.commons.errorhandling.types.ErrorColumn
+
+/**
+ * Class implement the functionality of filtering rows with columns.
+ */
+
+object ErrorHandlingFilterRowsWithErrors extends ErrorHandlingCommon {
+  /**
+   * Method return the rows that has errors on aggregation
+   * @param dataFrame  the overall data structure that need to be checked for rows with errors
+   * @param errCols  a final case class that provide error columns
+   * @return returns rows with errors
+   */
+  override def aggregateErrorColumns(dataFrame: DataFrame)(errCols: ErrorColumn*): DataFrame = {
```

Comment:
Don't need to override this one. It is a;ready implemented (exactly this way) in `ErrorHandlingCommon`.

Reply:
Okay, I'll get rid of that, thank you.